pkg/cmd/init/init.go

```diff
@@ -27,6 +27,19 @@ var (
 
 		# Initialize a new KCL project from local directory
 		kusion init /path/to/templates`
+
+	templatesShort = `Templates used to initialize a new project`
```

Comment:
It's a little confusing about the current expression, maybe you could add a verb like show or list at the beginning of this sentence. e.g. "List all templates used to initialize a new project"

Reply:
+1, and the subcommand `kusion init templates` is also a little confusing, a subcommand like`kusion init list-templates` sounds more clear